### PR TITLE
Added docs for framework.cache.prefix_seed

### LINKED ISSUE #7162: Document the cache.prefix_seed option
When you deploy multiple applications into the same path (on different servers) that share the same cache backend you need to configure the `cache.prefix_seed` option to prevent naming collisions.

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7162.

The contents are a copy+paste of the excellent comments by @xabbuh and @nicolas-grekas in the related issue.

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
an example: putting there the name of your application (provided you give each of your apps a unique name)

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1463,6 +1465,27 @@ throw
...
+
+If defined, this value is used as part of the "namespace" generated for the
+cache item keys. A common practice is to use the unique name of the application
+(e.g. ``symfony.com``) because that prevents naming collisions when deploying
+multiple applications into the same path (on different servers) that share the
+same cache backend.
+
+It's also useful when using `blue/green deployment`_ strategies and more
+generally, when you need to abstract out the actual deployment directory, for
+example when warming caches offline.
```
Comment (xabbuh) [inline reference/configuration/framework.rst]: What about enclosing "for example, when warming caches offline" with parentheses?

**Comment by xabbuh** [comment]
👍 

Status: Reviewed

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**